updates/commands.go

```diff
@@ -119,6 +123,111 @@ func createStatusMessage(user users.User, chatId int64, msg tgbotapi.MessageConf
 	return msg
 }
 
+func handleMonthlySummaryCommand(fatBotUpdate FatBotUpdate) {
+	update := fatBotUpdate.Update
+	bot := fatBotUpdate.Bot
+	originalChatId := update.FromChat().ID
+	user, err := users.GetUserFromMessage(update.Message)
+	if err != nil {
+		log.Error(err)
+		sentry.CaptureException(err)
+		return
+	}
+
+	if user.ID == 0 {
+		return
+	}
+	months := 6 // until we'll let the user enter the number of months
+	start, end := getDuration(months)
+	chatIds, err := user.GetChatIds()
+	if err != nil {
+		log.Error(err)
+		sentry.CaptureException(err)
+		return
+	}
+	if len(chatIds) > 0 {
+		for _, chatId := range chatIds {
+			createUserMonthlySummaryChart(bot, originalChatId, user, chatId, start, end)
+		}
+	} else {
+		log.Errorf("No chat IDs found for user %s", user.Username)
+	}
+}
+func createUserMonthlySummaryChart(bot *tgbotapi.BotAPI, originalChatId int64, user users.User, chatId int64, start, end time.Time) {
+	group, _ := users.GetGroup(chatId)
+	fileName := fmt.Sprintf("%s_%s.png", user.Username, group.Title)
+	months := getMonthsInRange(start, end)
+	userWorkoutsByMonthAsString := user.GetMonthlyWorkoutsSummary(group, start, end, months)
+	monthsStringSlice := "'" + strings.Join(months, "', '") + "'"
+	userWorkoutsByMonthStringSlice := strings.Join(userWorkoutsByMonthAsString, ", ")
+	chartConfig := createMonthlyChartConfig(monthsStringSlice, userWorkoutsByMonthStringSlice)
+	qc := createQuickChart(chartConfig)
+	file, err := os.Create(fileName)
+	if err != nil {
+		panic(err)
```

Comment:
`panic`ing is a bad idea :) it'll kill the process over not being able to create a file.
Better to send back an error and return.
Also it's generally conventional in Go to send back an `error` to be evaluated in the calling function, this is a good place to do that